internal/driver/driver_test.go

```diff
@@ -712,6 +715,73 @@ func cpuProfileSmall() *profile.Profile {
 	}
 }
 
+func cpuProfileNoFunctions() *profile.Profile {
```

Comment:
I may be missing something: the bug fix is about not properly handling multiple instances of the same function in a profile. Why do you add a profile with no functions here rather than extending some existing profile to trigger the issue? Why is it important to have a profile with no function information? It's good to have the number of test inputs to necessary minimum imo.

Reply:
I added a case with repeated functions.
The case with no functions is similar to the input on the issue.

Reply:
I think the "no functions" case is redundant even if it's similar to the input in the bug report since the root cause is not handling case when a function referenced by several locations. I'd remove it unless you think it has potential to expose some other bugs.

Reply:
Done. 
Did more testing on master, and current cpuProfile() has repeated functions, so can be used for this.